README.md

```diff
@@ -23,6 +23,18 @@ Quick Start Guide
 The server(s) can then be stopped with `python quickstart.py --stop`. This will
 only work if quickstart was used to start the server(s).
 
+Quick Start Guide (with docker)
+-------------------------------
+
+If you have Docker installed and working, you can do:
+
+1. `docker-compose build`
+2. `docker-compose up`
+
+Then use the sym server at http://localhost:8080 .
+
+You can stop the docker container by hitting `CTRL-C` twice.
+
```

Comment:
I think we should probably put `make build` and `make run` because the makefile seems to do more than just call `docker-compose`. When I tried just running the `docker-compose` commands, docker game me a couple Permission Denied errors.

Also, I feel like since we describe the installation process without Docker, perhaps we should describe it with Docker as well. I had a bit of a hard time figuring it out. I ended up needing these three sources to figure it out:
[https://docs.docker.com/engine/installation/linux/ubuntulinux/](https://docs.docker.com/engine/installation/linux/ubuntulinux/)
[https://docs.docker.com/compose/install/](https://docs.docker.com/compose/install/)
[http://stackoverflow.com/a/33596140/4103025](http://stackoverflow.com/a/33596140/4103025)